src/main/resources/portal.properties.EXAMPLE

```diff
@@ -277,4 +277,14 @@ ehcache.cache_type=none
 #hrrs.enable.logging=false
 
 # ensembl URL template for transcript lookup in Mutations tab. Default is http://grch37.ensembl.org/homo_sapiens/Transcript/Summary?t=<%= transcriptId %>
-#ensembl.transcript_url=http://ensembl.org/homo_sapiens/Transcript/Summary?t=<%= transcriptId %>
\ No newline at end of file
+#ensembl.transcript_url=http://ensembl.org/homo_sapiens/Transcript/Summary?t=<%= transcriptId %>
+
+# Enable/disable Persistent Cache (true, false)
+# The Persistent Cache is a frontend feature that is used to cache semi static
+# data produced by APIs. It uses IndexedDB as a means for persistent storage.
+# This feature is disabled by default.
+# enable_persistent_cache=true
+
+# Limit the size of gene queries (number). gene count * sample count < query_product_limit
+# This limit is enforced on the frontend.
+query_product_limit=1000000
```

Comment:
@alisman so following same logic as patient filter property, i guess for this case it might make sense to comment it out as well

Reply:
agreed. @Luke-Sikina sorry to ask you to change this again.  we've been thinking about configuration today and decided that many settings (e.g. this one) really aren't meant to be managed by installers except in edge cases.  by leaving these commented in the config examples, we can maintain control over the defaults in our own codebase.